src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -24,15 +24,21 @@ A simple migration file looks like this:
 Filename: `4_example_migration.js`
 
 ```javascript
-var MyContract = artifacts.require("MyContract");
+const MyContract = artifacts.require("MyContract");
 
-module.exports = function(deployer) {
+module.exports = async function (deployer, network, accounts) {
   // deployment steps
-  deployer.deploy(MyContract);
+  await deployer.deploy(MyContract);
 };
 ```
 
-Note that the filename is prefixed with a number and is suffixed by a description. The numbered prefix is required in order to record whether the migration ran successfully. The suffix is purely for human readability and comprehension.
+Note that the filename is prefixed with a number and is suffixed by a description. The numbered prefix is required in order to record whether the migration ran successfully. The suffix is purely for human readability and comprehension. Your migration function also may
```

Comment:
>The numbered prefix is required in order to record whether the migration ran successfully.

Is this correct?

Reply:
No, it has to do with the order in which it runs them I'm pretty sure. I didn't edit this sentence but I can update it.

Reply:
I added a bit about migration order.